package.json

```diff
@@ -29,9 +29,10 @@
     "bluebird": "^3.5.1",
     "d3": "^4.12.0",
     "immutable": "^3.8.2",
-    "np": "^2.20.1",
     "lodash.isequal": "^4.5.0",
-    "react-sizeme": "^2.4.3",
+    "np": "^2.20.1",
+    "react-select": "^1.2.1",
```

Comment:
it seems like an overhead to keep react-select inside the library as long as we use it only in stories, or webpack will handle that somehow?

Reply:
Webpack will not bundle it. It will start at src/index.js and not include what's exclusively used in stories.